Cargo.toml

```diff
@@ -6,13 +6,20 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+bstr = "1.1.0"
```

Comment:
I tried to mimic the exact version of the dependencies of `grep` but some of them look kind of old so might be fine to try and upgrade?